src/Microsoft.Health.Fhir.Api/Configs/ThrottlingConfiguration.cs

```diff
@@ -15,5 +15,9 @@ public class ThrottlingConfiguration
         public int ConcurrentRequestLimit { get; set; }
 
         public HashSet<ExcludedEndpoint> ExcludedEndpoints { get; } = new HashSet<ExcludedEndpoint>();
+
+        public int MaxQueueSize { get; set; }
+
+        public int MaxMillisecondsInQueue { get; set; }
```

Comment:
That's a good point. When we first brought this in, we decided not to include it in the appsettings because we didn't want to make recommendations on what the value should be. But I think we should document that this functionality exists. @brendankowitz what do you think?

Reply:
At first the numbers were fairly arbitrary and I think specific to some of the setups we had. I think I'd be ok with introducing a QoS guide. @CaitlinV39 thoughts?

Reply:
I've added comments on the configuration properties and included throttling in appsettings.json, with it disabled.